### PR TITLE
Update TestFixtureAttributeExamples.cs To give an example of using params in a constructor

### PR DESCRIPTION
**Pull request by Shiney**
see https://github.com/nunit/nunit/issues/1459 for when this was fixed

### CODE REVIEW

File: docs/snippets/Snippets.NUnit/Attributes/TestFixtureAttributeExamples.cs [hunk 1]
```diff
@@ -37,6 +39,17 @@ public ParameterizedTestFixture(int eq1, int eq2, int neq)
...
 
+        // Can use params arguments (but not yet optional arguments)
+        public ParameterizedTestFixture(params char[] eqArguments)
+        {
+            _eq1 = eqArguments[0].ToString();
+            _eq2 = eqArguments[1].ToString();
+            if (eqArguments.Length > 2)
+                _neq = eqArguments[2].ToString();
+            else 
+                _new = null;
```
[the remark this answers is not in the document]
Comment (SeanKilleen) [reply docs/snippets/Snippets.NUnit/Attributes/TestFixtureAttributeExamples.cs]: Fixed.

**Comment by SeanKilleen** [comment]
It appears this is failing. @OsirisTerje do I need to bump the NUnit version to fix this or am I missing something?

**Comment by OsirisTerje** [comment]
@Shiney Can you fix the tests please?   @SeanKilleen Don't believe this is the 4.0, but it should refer to 4.0.0-beta.1 for other examples.

**Comment by Shiney** [comment]
@OsirisTerje How is this done? The problem seems to be that the two extra uses of TestFixtureData only work with a very recently merged change on the main NUnit repository but the snippets are built against 4.0.0-beta.1 ?

**Comment by Shiney** [comment]
And there isn't a more recent beta nuget package

**Comment by OsirisTerje** [comment]
Then add an Ignore attribute to it, with a text saying that is waits for the next version :-)

**Comment by Shiney** [comment]
Hmm seems like it still fails to run the constructor regardless of any ignore attributes etc. on this, let's just wait until the next nuget package is used to progress this.

File: docs/snippets/Snippets.NUnit/Attributes/TestFixtureAttributeExamples.cs [hunk 2]
```diff
@@ -37,6 +39,17 @@ public ParameterizedTestFixture(int eq1, int eq2, int neq)
...
 
+        // Can use params arguments (but not yet optional arguments)
+        public ParameterizedTestFixture(params char[] eqArguments)
+        {
+            _eq1 = eqArguments[0].ToString();
+            _eq2 = eqArguments[1].ToString();
+            if (eqArguments.Length > 2)
+                _neq = eqArguments[2].ToString();
+            else 
+                _neq = null;
```
Comment (manfred-brands) [inline docs/snippets/Snippets.NUnit/Attributes/TestFixtureAttributeExamples.cs:50]: `_neq` declaration needs updating, append `?`, to allow assigning `null`

Comment (Shiney) [reply docs/snippets/Snippets.NUnit/Attributes/TestFixtureAttributeExamples.cs:50]: Fixing

**Comment by OsirisTerje** [comment]
@Shiney Did you  put the Ignore attribute on the class(Fixture) ?  If you do that, the ctor will NOT be called.

**Comment by Shiney** [comment]
Yes. I still had the error. The code for checking if a constructor exists that we can call and actually calling it are in different places, so presumably I'm getting that error instead.

**Comment by OsirisTerje** [comment]
@Shiney 4.0.0 and 4.0.1 is out now.  Can you check this PR again ?

**Comment by Shiney** [comment]
Thank you

**Pull request merged**